cmake/post.service.cmake

```diff
@@ -337,9 +337,18 @@ if(${ARCH} STREQUAL "x86_64")
   include_directories(${INSTALL_LOC}/${ARCH}/include)
 endif()
 
-add_library(libosdeps STATIC IMPORTED)
-set_target_properties(libosdeps PROPERTIES LINKER_LANGUAGE CXX)
-set_target_properties(libosdeps PROPERTIES IMPORTED_LOCATION ${INSTALL_LOC}/${ARCH}/lib/libosdeps.a)
+#add_library(libosdeps STATIC IMPORTED)
+#set_target_properties(libosdeps PROPERTIES LINKER_LANGUAGE CXX)
```

Comment:
Nice to get these split up - just delete the comments. You can always go back to git history if you need it back.